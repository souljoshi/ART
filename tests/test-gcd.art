int gcd(int a, int b)
{
    while(a != b)
    {
        if (a > b)
            a = a - b;
        else
            b = b -a;
    }
    return a;
}

int main()
{
    printi(gcd(7, 63));
    printi(gcd(2, 18));
    printi(gcd(3, 15));
    printi(gcd(1, 1));
    printi(gcd(100, 150));
}