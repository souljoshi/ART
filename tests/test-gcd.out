7
2
3
1
50
